homeassistant/components/nextbus/sensor.py

```diff
@@ -34,59 +36,50 @@
 )
 
 
-def validate_value(value_name, value, value_list):
-    """Validate tag value is in the list of items and logs error if not."""
-    valid_values = {v["tag"]: v["title"] for v in value_list}
-    if value not in valid_values:
-        _LOGGER.error(
-            "Invalid %s tag `%s`. Please use one of the following: %s",
-            value_name,
-            value,
-            ", ".join(f"{title}: {tag}" for tag, title in valid_values.items()),
+async def async_setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    async_add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
+    """Initialize nextbus import from config."""
+    async_create_issue(
+        hass,
+        DOMAIN,
+        "deprecated_yaml",
+        is_fixable=False,
+        breaks_in_ha_version="2023.7.0",
```

Comment:
```suggestion
        breaks_in_ha_version="2024.4.0",
```

We have a six month deprecation period